addon/components/nrg-dropdown/sub-menu.hbs

```diff
@@ -0,0 +1,18 @@
+<div
+  class="item {{if @disabled "disabled"}} {{if @active "active"}} {{@class}}"
+  data-dropdown-item={{@data-dropdown-item}}
```

Comment:
Where does `@data-dropdown-item` come from? I don't see it passed in from the `nrg-dropdown/menu` component.

Reply:
I don't think we use it in nrg-ui anywhere, but we use it in acceptance tests to be able to find and click a specific dropdown item. Fieldwork uses this functionality a lot in it's tests.-